src/runtime/auto-prompt-manager.ts

```diff
@@ -766,7 +767,8 @@ export class AutoPromptManager {
               isClosable,
               monetizationFunction: this.getLargeMonetizationPromptFn_(
                 autoPromptType,
-                !!isClosable
+                !!isClosable,
+                /* shouldNotAnimateFade */ true
```

Comment:
Read from `options`?

Reply:
This is for rewarded ads, so it should always suppress the animation here.